sycl/unittests/helpers/MockDeviceImage.hpp

```diff
@@ -312,12 +332,17 @@ class MockDeviceImage {
   /// - placeholder binary data
   MockDeviceImage(std::vector<MockOffloadEntry> &&OffloadEntries,
                   MockPropertySet PropertySet)
-      : MockDeviceImage(
-            SYCL_DEVICE_BINARY_VERSION, SYCL_DEVICE_BINARY_OFFLOAD_KIND_SYCL,
-            SYCL_DEVICE_BINARY_TYPE_SPIRV, __SYCL_DEVICE_BINARY_TARGET_SPIRV64,
-            "", "", {}, std::vector<unsigned char>{1, 2, 3, 4, 5},
-            internal::LifetimeExtender(std::move(OffloadEntries)),
-            std::move(PropertySet)) {}
+      : MockDeviceImage(SYCL_DEVICE_BINARY_VERSION,
+                        SYCL_DEVICE_BINARY_OFFLOAD_KIND_SYCL,
+                        SYCL_DEVICE_BINARY_TYPE_SPIRV,
+                        __SYCL_DEVICE_BINARY_TARGET_SPIRV64, "", "",
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
+                        {},
```

Comment:
```suggestion
                        {}, // Manifest.
```

I know it wasn't there before, but it's helpful.

Reply:
Well, this code is supposed to be removed in couple months, but I'll add it anyway :)